src/texture.cpp

```diff
@@ -145,9 +112,7 @@ void Texture::New(const int width, const int height)
         return;
     }
 
-    image = SDL_CreateRGBSurface(SDL_SWSURFACE, width, height, 32,
-                                 RGBA8Format.Rmask, RGBA8Format.Gmask,
-                                 RGBA8Format.Bmask, RGBA8Format.Amask);
+    image = SDL_CreateRGBSurfaceWithFormat(0, width, height, 32, SDL_PIXELFORMAT_RGBA8888);
```

Comment:
Is `image` leaked here? The new surface does not seem to be deallocated anywhere!